packages/cloud_firestore/cloud_firestore/ios/cloud_firestore.podspec

```diff
@@ -24,17 +24,17 @@ Pod::Spec.new do |s|
   s.license          = { :file => '../LICENSE' }
   s.authors          = 'The Chromium Authors'
   s.source           = { :path => '.' }
-  
+
   s.source_files     = 'Classes/**/*.{h,m}'
   s.public_header_files = 'Classes/Public/*.h'
   s.private_header_files = 'Classes/Private/*.h'
 
-  s.ios.deployment_target = '10.0'
+  s.ios.deployment_target = '11.0'
```

Comment:
We should not forget to make this a breaking change

Reply:
It's needed anyway since Firestore is dependant on 11.0 https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseFirestore.podspec

Reply:
This will be updated in the iOS SDK PR: https://github.com/firebase/flutterfire/pull/9708